src/styles/main.scss

```diff
@@ -1,3 +1,44 @@
 body {
   margin: 0;
+  counter-reset: my-counter;
+  
+}
+
+$cell-size: 100px;
+$base-color: #eee;
+
+.calendar {
+  box-sizing: border-box;
+  border: 2px solid cornflowerblue;
+  width: calc(7 * #{$cell-size});
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0;
+  padding: 0;
+
+  &__day {
+    display: flex;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    box-sizing: border-box;
+    border: 1px solid #000;
+    height: $cell-size;
+    width: $cell-size;
+    color: #000;
+    list-style: none;
+    align-items: center;
+    justify-content: center;
+    margin: 1px;
+    
+  
+  }
+  @for $index from 1 through 31 {
```

Comment:
use this loop for numbering the dates
for background color use `calender__day` class

Reply:
I struggle to get the loop right. The numbers seem impossible to appear.

Reply:
@for $index from 1 through 31 {
      &_:nth-child(#{$index})::before {
        content: "#{$index}";
      }
    }
  }

Reply:
remove` _ `after `&`
```
@for $index from 1 through 31 {
  &:nth-child(#{$index})::before {
    content: "#{$index}";
  }
}
```